app-guides/mysql-on-fly.html.md

```diff
@@ -151,7 +152,7 @@ flyctl proxy 13306:3306 -a my-mysql
 Then connect to your MySQL server at `localhost:3306` and the username and password credentials from above:
 
 ```cmd
-mysql -h localhost -P 3306 -u hotdogs_are_tacos -ppassword cube_theory
+mysql -h localhost -P 3306 -u hotdogs_are_tacos -ppassword some_db
```

Comment:
change hotdocs_are_tacos to non_root_user as well here, as changed in line 70 above

Reply:
also! just a quick point out the title of the current branch is for Symfony changes